planner/cardinality/selectivity_test.go

```diff
@@ -724,34 +535,6 @@ func TestDNFCondSelectivity(t *testing.T) {
 	testKit.MustExec("explain select * from tt where col1 is not null or col2 not between 454623814170074.2771 and -975540642273402.9269 and col3 not between '2039-1-19 10:14:57' and '2002-3-27 14:40:23';")
 }
 
-func TestIndexEstimationCrossValidate(t *testing.T) {
-	store := testkit.CreateMockStore(t)
-	tk := testkit.NewTestKit(t, store)
-	tk.MustExec("use test")
-	tk.MustExec("drop table if exists t")
-	tk.MustExec("create table t(a int, b int, key(a,b))")
-	tk.MustExec("insert into t values(1, 1), (1, 2), (1, 3), (2, 2)")
-	tk.MustExec("analyze table t")
-	require.NoError(t, failpoint.Enable("github.com/pingcap/tidb/statistics/table/mockQueryBytesMaxUint64", `return(100000)`))
```

Comment:
How is failpoint handled?

Reply:
This test (`TestIndexEstimationCrossValidate`) is actually not moved. I have checked with diff that I didn't move any test containing a failpoint 🤔 .